R/gh.R

```diff
@@ -273,6 +276,10 @@ gh_make_request <- function(x, error_call = caller_env()) {
     )
   }
 
+  if (!is.null(x$max_rate)) {
+    req <- httr2::req_throttle(req, x$max_rate)
```

Comment:
Should we make an attempt to create some meaningful `realm` here? Otherwise packages might interfere.

Reply:
The realm is automatically taken from the hostname, which I think should be correct.

Reply:
Right!